skyportal/facility_apis/ztf.py

```diff
@@ -206,6 +363,123 @@ def delete(request):
 
         DBSession().add(transaction)
 
+    @staticmethod
+    def get(request):
+
+        """Get a forced photometry request result from ZTF.
+
+        Parameters
+        ----------
+        request : skyportal.models.FollowupRequest
+            The request to add to the queue and the SkyPortal database.
```

Comment:
I'm not sure this doc string is the right one...

Reply:
always tough.